p2p/README.md

```diff
@@ -4,7 +4,7 @@ The p2p package provides an abstraction around peer-to-peer communication.
 
 Docs:
 
-- [Connection](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/connection.md) for details on how connections and multiplexing work
-- [Peer](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/node.md) for details on peer ID, handshakes, and peer exchange
-- [Node](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/node.md) for details about different types of nodes and how they should work
-- [Config](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/config.md) for details on some config option
+- [Connection](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/legacy-docs/connection.md) for details on how connections and multiplexing work
+- [Peer](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/legacy-docs/node.md) for details on peer ID, handshakes, and peer exchange
+- [Node](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/legacy-docs/node.md) for details about different types of nodes and how they should work
+- [Config](https://github.com/cometbft/cometbft/blob/v0.38.x/spec/p2p/legacy-docs/config.md) for details on some config option
```

Comment:
Same as https://github.com/cometbft/cometbft/pull/1004#discussion_r1235564777

Reply:
Removed full links in https://github.com/cometbft/cometbft/pull/1005/commits/58777000c2aeb8fa09f3a40b52da740097a7aeac